src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/code/RegisterConfig.java

```diff
@@ -69,7 +69,7 @@ default int getMaximumFrameSize() {
      * @param type the type of calling convention
      * @param kind specifies what kind of registers is being requested
      * @return the ordered set of registers that may be used to pass parameters in a call conforming
-     *         to {@code type}
+     * to {@code type}
```

Comment:
What happened to the formatting? The second line of the `@return` clause should be indented.